go/chat/inboxsource.go

```diff
@@ -414,15 +409,18 @@ func (s *localizer) localizeConversation(ctx context.Context, uid gregor1.UID,
 		return chat1.ConversationLocal{Error: &errMsg}
 	}
 
-	info, err := utils.LookupTLF(ctx, s.getTlfInterface(), conversationLocal.Info.TlfName, conversationLocal.Info.Visibility)
-	if err != nil {
-		errMsg := err.Error()
-		return chat1.ConversationLocal{Error: &errMsg}
+	// Only do this check if there is a chance the TLF name might be an SBS name.
+	if strings.Contains(conversationLocal.Info.TlfName, "@") {
```

Comment:
follow up:  there's a lot of code in libkb/assertion.go that we should probably use instead of strings.Contains("@").